### PR TITLE
Add airnode-deployer `rollback` command

### LINKED ISSUE #1167: Add rollback functionality for Airnode deployments
Documentation for https://github.com/api3dao/airnode/issues/1406

### PR DESCRIPTION
**Pull request by amarthadan**
Close #1167

### CODE REVIEW

File: docs/airnode/v0.10/reference/packages/deployer.md [hunk 1]
```diff
@@ -217,6 +217,28 @@ Options:
 airnode-deployer info aws2c6ef2b3
 ```
 
+### Reverting to a previous version
+
+In case you want to revert to one of the previous versions of your deployment,
+you can do so with the `rollback` command.
+
+### rollback
```
Comment (dcroote) [inline docs/airnode/v0.10/reference/packages/deployer.md]: nit: command heading should be additional level 
```suggestion
#### rollback
```

File: docs/airnode/v0.10/reference/packages/deployer.md [hunk 2]
```diff
@@ -217,6 +217,28 @@ Options:
...
+
+```bash
+# Deploy one of the previous Airnode deployment versions
+
+Positionals:
+  deployment-id  ID of the deployment (from 'list' command)                                          [string] [required]
+  version-id     ID of the deployment version (from 'info' command)                                  [string] [required]
```
Comment (dcroote) [inline docs/airnode/v0.10/reference/packages/deployer.md]: I wouldn't say it's critical, but the `version-id` argument would be clearer if it were slightly more explicit:

ID of the deployment version **to rollback to**

(Applies to `api3dao/airnode` as well...)

**Review by dcroote — APPROVED** [review]
Tested and it works. Two small nits

File: docs/airnode/v0.10/reference/packages/deployer.md (hunk 2 above)
Comment (amarthadan) [reply docs/airnode/v0.10/reference/packages/deployer.md]: Right, I'll add it :slightly_smiling_face:

Comment (amarthadan) [reply docs/airnode/v0.10/reference/packages/deployer.md]: I've updated the help messages for all the commands and added a commit to this PR. Hope you don't mind.

**Review by dcroote — APPROVED** [review]
Quick turnaround from the Airnode PR! LGTM

**Pull request merged**